org.eclipse.xtext.xbase.ui.testing/src/org/eclipse/xtext/xbase/ui/testing/AbstractXbaseContentAssistTest.java

```diff
@@ -53,6 +55,82 @@
  */
 public abstract class AbstractXbaseContentAssistTest extends Assert implements ResourceLoadHelper, IJavaProjectProvider {
 
+	private static final boolean isJava11OrLater = determineJava11OrLater();
+
```

Comment:
In general, I think these utility methods might be useful also in other tests, why not make them available somewhere?

Reply:
see https://github.com/eclipse/xtext/issues/1452